utils/smtp_client.py

```diff
@@ -78,9 +78,14 @@ def send_mail(name, subject, body, settings=None):
 
     msg = MIMEMultipart()
     from_name = str(Header('App SRE team automation', 'utf-8'))
-    to = '{}@{}'.format(name, _mail_address)
     msg['From'] = formataddr((from_name, _username))
-    msg['To'] = to
+    to = set()
+    for name in names:
+        if '@' in name:
+            to.add(name)
+        else:
+            to.add(f"{name}@{_mail_address}")
+    msg['To'] = ', '.join(to)
```

Comment:
ok

Reply:
What's the differentece now between `send_mail` and `send_mails`? Do we still need `send_mails` after this?

Reply:
`send_mail` sends a single mail to (now) multiple recipients.
`send_mails` sends multiple mails to (now) multiple recipients.

so still both needed. The reason we want to send multiple mails is because the `send_mails` function is used by the `terraform-users` integration, which sends email invitations to our AWS accounts, and those mails contain a GPG encrypted password for that specific user. so these mails must be sent separately. hope that makes sense!